json-schema/card-flattened-schema.json

```diff
@@ -259,6 +259,10 @@
                 "description": "A shortcode representing the rarity the card printing was printed at.",
                 "type": "string"
             },
+            "expansion_slot": {
+                "description": "Whether the card is an expansion slot card or not.",
```

Comment:
### ⚠️ Suggestion, blocking

"Whether the card printing comes in the expansion slot or not" might be more accurate (and reflects that the printing is what comes in the expansion slot, not the card itself)

Same request for the card-schema.json 😄

Reply:
I also changed it in csv-schemas.md. Fixed.